SPECS/iptables/iptables.spec

```diff
@@ -18,7 +18,8 @@ BuildRequires:  libmnl-devel
 BuildRequires:  libnftnl-devel
 BuildRequires:  systemd-bootstrap-rpm-macros
 Requires:       iana-etc
-Requires:       systemd
+# Our build tooling cannot handle this
+#Requires:       systemd
```

Comment:
This looks worrying. What is exactly the issue here? If `systemd` is a real run-time dependency of this package, then we must still keep it.

Reply:
this is because of our "circular dependencies". I actually don't understand myself why our toolkit fails around runtime dependencies, as it's simply *building* the package, but it does indeed fail when attempting to build package A if there is a "circular dependency" - even *runtime* dependency - on package B. I'm open to alternate suggestions, although I do plan to look into a more comprehensive actual fix separately from this pr.

Reply:
also, i'm not aware of *any* situation where our distro runs without systemd installed, are you?